src/Nox.Types.EntityFramework/Types/Year/YearConverter.cs

```diff
@@ -4,5 +4,5 @@ namespace Nox.Types.EntityFramework.Types;
 
 public class YearToUShortConverter : ValueConverter<Year, ushort>
```

Comment:
Can we rename the converter to "YearConverted"?

Reply:
Done